x11-terms/terminology/terminology-1.0.0.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2017 Gentoo Foundation
```

Comment:
The `Gentoo-Bug` commit tag is obsolete. Please use either `Bug` or `Closes` (with full URL), depending on whether you wish to close a bug or just leave a comment.

Reply:
Done